<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finproceso</class>
 <widget class="QMainWindow" name="finproceso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS UI Gothic</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PET-EATER</string>
  </property>
  <property name="windowIcon">
   <iconset theme="PET">
    <normaloff>:/icono/PET.png</normaloff>:/icono/PET.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="fondo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	
	
	background-color: rgb(212, 236, 212);
	border-radius:20px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>21</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;BOTELLAS &lt;/p&gt;&lt;p&gt;QUE INGRESÓ:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>300</y>
       <width>341</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>21</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;BOTELLAS TOTALES&lt;/p&gt;&lt;p&gt;INGRESADAS&lt;/p&gt;&lt;p&gt;EN EL DÍA:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="ntbotellas">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>460</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>10</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>21</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;GRACIAS POR RECICLAR&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>461</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>21</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tengo algo para ti, ¡escaneame!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/imagenes/aplauso.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>380</y>
       <width>120</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/imagenes/PET.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="regresar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(167, 186, 167);
image: url(:/imagenes/regresar.png);
background-color: rgb(168, 186, 168);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="qr">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>100</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>5</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imag.qrc"/>
 </resources>
 <connections/>
</ui>
